caproto/_commands.py

```diff
@@ -387,33 +387,31 @@ def read_from_bytestream(data, role):
 _commands = set()
 
 
-class _MetaDirectionalMessage(type):
-    # see what you've done, @tacaswell and @danielballan?
```

Comment:
It makes me sad to remove this line, but moving from metaclass to `__init_subclass__` is for the best, given that we have a reason to need class kwargs.